packages/clarity/clarity.0.1.4/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+version: "0.1.4"
+maintainer: "joyprophecy@gmail.com"
+author: "joyprophecy@gmail.com"
+homepage: "https://github.com/IndiscriminateCoding/clarity"
+bug-reports: "https://github.com/IndiscriminateCoding/clarity/issues"
+dev-repo: "https://github.com/IndiscriminateCoding/clarity.git"
+license: "BSD3"
+available: [ ocaml-version >= "4.04.0" ]
+depends: [ "jbuilder" {build} "ocamlfind" {build} ]
+build: [
+  [ "jbuilder" "build" ]
+]
+install: [
+  [ "jbuilder" "install" ]
```

Comment:
it's not a blocker to merge, but i think that `install` and `remove` are not needed, since `jbuilder` generates a `.install` file for you.